Pegasus/include/Geometry.hpp

```diff
@@ -401,18 +407,24 @@ inline void Initialize<Plane, Box>(SimpleShape const* a, SimpleShape const* b, C
     auto cache = static_cast<Cache<Plane, Box>*>(cacheBase);
 
     cache->planeNormal = plane->GetNormal();
-    cache->planeDistance = glm::dot(plane->getCenterOfMass(), cache->planeNormal);
+    cache->planeDistance = glm::dot(plane->GetCenterOfMass(), cache->planeNormal);
 
     box->GetAxes(cache->boxAxes[0], cache->boxAxes[1], cache->boxAxes[2]);
-    cache->boxMassCenter = box->getCenterOfMass();
+    cache->boxMassCenter = box->GetCenterOfMass();
     CalculateBoxVertices(cache->boxAxes[0], cache->boxAxes[1], cache->boxAxes[2], cache->boxVertices.begin());
     cache->boxFaces = {
         cache->boxAxes[0], cache->boxAxes[1], cache->boxAxes[2],
         cache->boxAxes[0] * -1.0, cache->boxAxes[1] * -1.0, cache->boxAxes[2] * -1.0
     };
-    std::for_each(cache->boxVertices.begin(), cache->boxVertices.end(), [cache](auto& n) { n += cache->boxMassCenter; });
+    std::for_each(cache->boxVertices.begin(), cache->boxVertices.end(), [cache](auto& n)
```

Comment:
Nope, accumulate will calculate the sum of the vertices, but this code performs transformation by translation.